spec/factories/users.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :user do
+    
+  end
+end
```

Comment:
No attributes in the factory?

Reply:
We have addressed your insights. Created Factory Bot with email and password.